app/models/services/base-service.ts

```diff
@@ -159,6 +162,16 @@ function stepRequired(step: ServiceStep, operation: DataOperation): boolean {
       required = true;
     }
   }
+  if (required && step.conditional?.umm_c) {
+    required = false;
```

Comment:
I think the default should be the step is required - so move this line to set it to false below the `if (step.conditional.umm_c.native_format)` check.